libs/ui/src/lib/feedback/feedback/feedback.component.html

```diff
@@ -1,6 +1,7 @@
 <cvi-ng-radio-group [appearance]="appearance"
                     [title]="title"
-                    [formControl]="radioButtonsControl">
+                    [formControl]="radioButtonsControl"
+                    style="--radio-group-font-size: 12px">
```

Comment:
styles should not be written in html

Reply:
@siimveskilt I'd say in this case it is OK. We discussed this with @ekateriinal, and turning the font size into a CSS variable that can be set outside of the component seemed like a good option. We already use this kind of external styling API in `cvi-tooltip` or `cvi-textfield`.

Reply:
it is a non-functional requirement that there are no styles used directly inside html files